### PR TITLE
Dev branch

### PR DESCRIPTION
**Pull request by freerangeunicorn**
@schremsn 

- [ ] code review

### CODE REVIEW

**Comment by freerangeunicorn** [comment]
readme.me and public in master not needed

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+	// List of extensions which should be recommended for users of this workspace.
+	"recommendations": [
+		
+	],
+	// List of extensions recommended by VS Code that should not be recommended for users of this workspace.
+	"unwantedRecommendations": [
+		
+	]
```
Comment (schremsn) [inline .vscode/extensions.json:12]: I recommend adding this file gitignore

File: package.json [hunk 2]
```diff
@@ -3,13 +3,18 @@
...
+    "@fortawesome/react-fontawesome": "^0.1.11",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "@wojtekmaj/react-timerange-picker": "^3.0.0",
     "bootstrap": "^4.5.0",
     "react": "^16.13.1",
     "react-bootstrap": "^1.0.1",
     "react-bootstrap-timezone-picker": "^2.0.1",
+    "react-datepicker": "^3.1.3",
```
Comment (schremsn) [inline package.json:17]: are you still using both date and timerange picker?

Comment (freerangeunicorn) [reply package.json:17]: Yes for the teacher to create a class within a certain time range

File: .vscode/extensions.json (hunk 1 above)
Comment (freerangeunicorn) [reply .vscode/extensions.json:12]: Will add it

File: src/components/StudentProfile.js [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+  in30Days.setDate(in30Days.getDate() + 30);
+  const [time, setTime] = useState();
+
+  console.log("show me data; yoga class", yogaClass);
+  
+  const handleChange = (date) => {
+    setDate(date);
+    let hours = date.getHours();
+    let minutes = date.getMinutes();
+    let day = date.getDay();
```
Comment (schremsn) [inline src/components/StudentProfile.js:45]: you are not using the day

File: src/components/StudentProfile.js [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+                    <ListGroup.Item> Time: {yogaClass.time}</ListGroup.Item>
+                    <ListGroup.Item>Price: ${yogaClass.price}</ListGroup.Item>
+                    <ListGroup.Item>Date: {yogaClass.date}</ListGroup.Item>
+                    <ListGroup.Item>Style: {yogaClass.style}</ListGroup.Item>
+                    <ListGroup.Item>
+                      Duration: {yogaClass.duration} minutes
+                    </ListGroup.Item>
+                  </ListGroup>
+                  <Card.Text> Description: {yogaClass.description}</Card.Text>
+                  <Button variant="dark" onClick={() => handleSearch()}></Button>  {/* can u use the same book for async? */}
```
Comment (schremsn) [inline src/components/StudentProfile.js:310]: What does this button do? you are calling search again?

File: src/components/TeacherProfile.js [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+  const [time, setTime] = useState();
+  const onChange = (time) => setTime(time);
+  // console.log(time)
+  const [yogaClass, setYogaClass] = useState([]);
+  const [teacherFirstName, setTeacherFirstName] = useState();
+  const [teacherLastName, setTeacherLastName] = useState();
+  const [teacherEmail, setTeacherEmail] = useState();
+  const [teacherExperience, setTeacherExperience] = useState();
+
+  useEffect(() => {
```
Comment (schremsn) [inline src/components/TeacherProfile.js:48]: Look into refactoring to use getClasses

**Review by schremsn — APPROVED** [review]
All up start adding comments, to make it easier to understand the code flow

File: src/components/StudentProfile.js (hunk 4 above)
Comment (freerangeunicorn) [reply src/components/StudentProfile.js:310]: Calling book

**Pull request closed without merging**